torch/csrc/jit/passes/quantization/dedup_module_uses.h

```diff
@@ -10,6 +10,17 @@ namespace jit {
  *  the type will be the same as before and all the attributes will be
  *  copied, then we'll change the use of the original module to the use
  *  of cloned module in the Graph.
+ *
+ *  This is done to ensure that modules can survive destructive passes
+ *  without changing model behavior. For example, here:
+ *
+ *    x = self.conv1(x)
+ *    x = self.relu(x)
+ *    x = self.conv2(x)
+ *    x = self.relu(x)
+ *
+ *  self.relu needs to be deduplicated for conv-bn folding to work
```

Comment:
we don't fold relu in conv bn folding, probably bn is a better example.

Reply:
there is little reliance on this pass as we change the implementation now I think, maybe we can remove this pass at some point

Reply:
probably bn module will not be used multiple times